contracts/connectors/reflexer/events.sol

```diff
@@ -0,0 +1,26 @@
+pragma solidity ^0.7.0;
+
+contract Events {
+    event LogOpen(uint256 indexed safe, bytes32 indexed collateralType);
+    event LogClose(uint256 indexed safe, bytes32 indexed collateralType);
+    event LogTransfer(uint256 indexed safe, bytes32 indexed collateralType, address newOwner);
+    event LogDeposit(uint256 indexed safe, bytes32 indexed collateralType, uint256 tokenAmt, uint256 getId, uint256 setId);
+    event LogWithdraw(uint256 indexed safe, bytes32 indexed collateralType, uint256 tokenAmt, uint256 getId, uint256 setId);
+    event LogBorrow(uint256 indexed safe, bytes32 indexed collateralType, uint256 tokenAmt, uint256 getId, uint256 setId);
+    event LogPayback(uint256 indexed safe, bytes32 indexed collateralType, uint256 tokenAmt, uint256 getId, uint256 setId);
+    event LogWithdrawLiquidated(uint256 indexed safe, bytes32 indexed collateralType, uint256 tokenAmt, uint256 getId, uint256 setId);
+    event LogExit(uint256 indexed safe, bytes32 indexed collateralType, uint256 tokenAmt, uint256 getId, uint256 setId);
+    event LogDepositCoin(uint256 tokenAmt, uint256 getId, uint256 setId);
+    event LogWithdrawCoin(uint256 tokenAmt, uint256 getId, uint256 setId);
+
```

Comment:
```suggestion

```

we can remove these two events, since we are not using `LogDepositCoin` and `LogWithdrawCoin` anywhere.